lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1929,10 +1929,9 @@ public function get_collection_params() {
 	 *
 	 * @param  mixed $value
 	 * @param  WP_REST_Request $request
-	 * @param  string $parameter
 	 * @return WP_Error|boolean
 	 */
-	public function validate_user_can_query_private_statuses( $value, $request, $parameter ) {
+	public function validate_user_can_query_private_statuses( $value, $request ) {
```

Comment:
This function is consumed by the attachments controller as well, and matches the signature of other validation methods. As the parameter will be passed in to this method even though it is not used, I believe it is appropriate to leave the parameter in place. (@joehoyle or @rmccue, please disagree if I am off-base here).